README.md

```diff
@@ -1190,7 +1190,26 @@ try {
 } catch (DescopeException de) {
     // Handle the error
 }
+```
+Generate a JWT for a user, simulating a signin request.
 
+```java
+JwtService jwts = descopeClient.getManagementServices().getJwtService();
+AuthenticationInfo res = jwts.mgmt.jwt.signIn("dummy");
```

Comment:
I think that's wrong - why do you have `jwts.mgmt.jwt.signIn`? it should be `jwts.signIn("dummy", null)` I believe